test/Interop/Cxx/class/method/methods.swift

```diff
@@ -11,6 +11,8 @@ CxxMethodTestSuite.test("() -> Void") {
   var instance = HasMethods()
 
   instance.nonConstMethod()
+  instance.nonConstMethod(5)
+  HasMethods.nonConstMethod(4.2) // Testing name collision
```

Comment:
Could you please also add a test with const methods, e.g.
```
static void constMethod();
void constMethod() const;
```
I think it might reproduce the original issue.

Reply:
Indeed! This did reproduce the issue! Thanks for pointing this out. I updated the PR to fix the problem.